app/src/main/java/com/codingblocks/cbonlineapp/PreferenceManager.kt

```diff
@@ -0,0 +1,35 @@
+package com.codingblocks.cbonlineapp
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.content.SharedPreferences
+
+class PreferenceManager {
```

Comment:
can you explain why you created a new class?

Reply:
Although could have done  using PreferenceHelper , but for having clarity about clearing sharedprefs I created new class PreferenceManager

Reply:
@aggarwalpulkit596

Reply:
im not sure if you should clear a seperate class for handling preferences. all you need is the clear prefs. all the rest is useless, you should copy the function in the utils and use it

Reply:
not needed use preference helper